tests/manual-test-cases/Group5-Functional-Tests/5-25-OPS-User-Grant.robot

```diff
@@ -17,8 +17,16 @@ Documentation  Test 5-25 - OPS-User-Grant
 Resource  ../../resources/Util.robot
 Suite Setup  Wait Until Keyword Succeeds  10x  10m  Ops User Create
 Suite Teardown  Run Keyword And Ignore Error  Nimbus Cleanup  ${list}
+Test Teardown  Run Keyword And Ignore Error  Gather VC Logs
 
 *** Keywords ***
+
+Gather VC Logs
```

Comment:
This should go into one of the utility libraries.
I'm also unsure whether it will download the ESX log bundles or just VC - a keyword comment noting which would be good.